packages/web3-utils/src/formatter.ts

```diff
@@ -174,7 +175,29 @@ export const convert = (
 
 		// If value is an array
 		if (Array.isArray(value)) {
-			if (isNullish(schemaProp?.items)) {
+			let _schemaProp = schemaProp;
+
+			// TODO This is a naive approach to solving the issue of
+			// a schema using oneOf. This chunk of code was intended to handle
+			// BlockSchema.transactions
+			// TODO BlockSchema.transactions are not being formatted
```

Comment:
Our formatter should probably enter an array of transaction object and format each one, but currently isn't

Reply:
Created #5370 to track this